test/UserCodeRunner.spec.ts

```diff
@@ -634,13 +634,39 @@ describe('behavior', () => {
       Error: This is a test error
           at additionalFile:1:7
           at SourceTextModule.evaluate (node:internal/vm/module:224:23)
-          at UserCodeRunner.executeUserCode (/Users/jdstewar/gitRepos/jpl/mpcs/aerie/aerie-ts-user-code-runner/src/UserCodeRunner.ts:232:24)
+          at UserCodeRunner.executeUserCodeFromArtifacts (/Users/jdstewar/gitRepos/jpl/mpcs/aerie/aerie-ts-user-code-runner/src/UserCodeRunner.ts:222:24)
```

Comment:
I noticed these hard-coded values. This will cause the test to fail on my laptop as my user name is `rrgoetz` Not a deal breaker.

Reply:
Good point. I can refine the check here.